nuxeo-runtime/nuxeo-runtime-stream/src/test/java/org/nuxeo/runtime/stream/tests/TestAvroSchemaStoreService.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     pierre
+ */
+package org.nuxeo.runtime.stream.tests;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import javax.inject.Inject;
+
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaNormalization;
+import org.apache.avro.message.SchemaStore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.runtime.api.Framework;
+import org.nuxeo.runtime.avro.AvroSchemaStoreService;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.RuntimeFeature;
+
+/**
+ * @since 10.2
+ */
+@RunWith(FeaturesRunner.class)
+@Features(RuntimeFeature.class)
+@Deploy("org.nuxeo.runtime.stream")
+@Deploy("org.nuxeo.runtime.stream:test-avro-contrib.xml")
+public class TestAvroSchemaStoreService {
+
+    @Inject
+    public AvroSchemaStoreService service;
+
+    @Test
+    public void testSchemaStoreRetrieval() {
+        assertNotNull(Framework.getService(SchemaStore.class));
+        assertNotNull(Framework.getService(AvroSchemaStoreService.class));
```

Comment:
In fact this test is not useful, the framework will fail to start if the injection can't be processed.

Reply:
Yes, made it like this for consistency, since I don't inject the SchemaStore itself I asserted the retrieval and put the other altogether.
Just removed the second one.
👍